internal/ext/importer.go

```diff
@@ -47,6 +47,14 @@ func (i *Importer) Import(ctx context.Context, r io.Reader) error {
 		return fmt.Errorf("unmarshalling document: %w", err)
 	}
 
+	if doc.Version != "" && doc.Version != "1.0" {
+		return fmt.Errorf("unsupported version: %s", doc.Version)
+	}
+
+	if doc.Namespace != "" && i.namespace != "" && doc.Namespace != i.namespace {
+		return fmt.Errorf("namespace mismatch: %s != %s", doc.Namespace, i.namespace)
+	}
```

Comment:
I guess it will be up to the user to decide what they would want to do... Either remove the namespace from the document or do not supply the argument on the flag set. Do we have a preference for which they should do? If so what do you think we about curating the error message towards that preference?

Reply:
good point. im not sure if we have a preference, however I will try to come up with a friendlier error message